src/_cffi_src/openssl/ssl.py

```diff
@@ -703,4 +722,8 @@
 static const long TLS_ST_BEFORE = 0;
 static const long TLS_ST_OK = 0;
 #endif
+
+Cryptography_STACK_OF_SSL_CIPHER * Cryptography_get_ssl_session_ciphers(const SSL_SESSION *s) {
```

Comment:
`SSL_SESSION` is entirely opaque in OpenSSL 1.1.0 so we need a different way to get this information. It looks like CPython uses `SSL_get_ciphers` in master (https://github.com/python/cpython/blob/4106e4237df5397efc785ce4daa257aecb5bff75/Modules/_ssl.c#L1650), although I'm not 100% sure this is what you're attempting to mirror.